docs/docs/features/split-keyboards.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Split Keyboards
+sidebar_label: Split Keyboards
+---
+
+ZMK supports split keyboard setups where two or more keyboard parts (also called "sides" or "halves"), each with their own controller running ZMK, communicate with each other to work as a single keyboard device.
+
+:::note[Split communication protocols]
+Currently ZMK only supports split keyboards that communicate with each other wirelessly over BLE.
+As such, only controllers that support BLE can be used with ZMK split keyboards.
+
+Supporting split communication over wired protocols is planned, which would allow using ZMK split keyboards with non-wireless controllers.
+:::
+
+## Central and Peripheral Roles
+
+In ZMK splits, one keyboard part has the "central" role that receives key position and sensor events from the other parts that are called "peripherals."
+This part runs the necessary keymap logic to convert received events into HID events such as keycodes and then communicates with the connected host devices, e.g. over USB or bluetooth.
+
+As a result of this architecture the internal keyboard state (like active layers) is shared and is mainly handled by the central part.
+However this also means that the peripheral parts _cannot_ communicate with host devices on their own, since they can only communicate with the central part.
+They will not present as keyboard devices when connected over USB and will not advertise as pairable BLE keyboards.
+
+:::note[Latency considerations]
+Since peripherals communicate through centrals, the key and sensor events originating from them will naturally have a larger latency, especially with a wireless split communication protocol.
+:::
```

Comment:
I think that makes sense. In that case I should convert this to a subsection perhaps?

Reply:
I think that would be nice, yeah.

Reply:
I added a section but I omitted any references to settings because
- these are ZMK internal Kconfig symbols that currently aren't documented in the config section
- they are already set for minimum possible latency so I didn't see much point in giving pointers to change them